src/main/java/io/cdap/e2e/utils/ElementHelper.java

```diff
@@ -0,0 +1,169 @@
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.e2e.utils;
+
+import org.junit.Assert;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Element helper
+ */
+public class ElementHelper {
+  private static final Logger logger = LoggerFactory.getLogger(ElementHelper.class);
+  private static final JavascriptExecutor js = (JavascriptExecutor) SeleniumDriver.getDriver();
+  private static final Actions actions = new Actions(SeleniumDriver.getDriver());
+
+  public static void scrollToElement(WebElement element) {
+    logger.info("Scrolling to the element: " + element);
+    actions.moveToElement(element).perform();
+  }
+
+  /**
+   * Scrolling to get the element in view using the Javascript Executor.
+   * It should only be used if necessary, as it does not replicate how a user would interact with the page.
+   * @param element
+   */
+  public static void scrollToElementUsingJsExecutor(WebElement element) {
+    logger.info("Scrolling to the element: " + element + " using the JsExecutor");
+    js.executeScript(ConstantsUtil.JS_SCROLL_INTO_VIEW, element);
+  }
+
+  /**
+   * Clicking on an element using the Javascript Executor.
+   * It should only be used if necessary, as it does not replicate how a user would interact with the page.
+   * @param element
+   */
+  public static void clickOnElementUsingJsExecutor(WebElement element) {
+    logger.info("Click on the element using Javascript Executor: " + element);
+    js.executeScript(ConstantsUtil.JS_CLICK, element);
+  }
+
+  public static void clickOnElement(WebElement element) {
+    WaitHelper.waitForElementToBeClickable(element);
+    scrollToElement(element);
+    logger.info("Click on the element: " + element);
+    element.click();
+  }
+
+  public static void clickOnElement(WebElement element, long timeoutInSeconds) {
+    WaitHelper.waitForElementToBeClickable(element, timeoutInSeconds);
+    scrollToElement(element);
+    logger.info("Click on the element: " + element);
+    element.click();
+  }
+
+  public static void clickIfDisplayed(WebElement element, long timeoutInSeconds) {
+    if (isElementDisplayed(element, timeoutInSeconds)) {
+      clickOnElement(element, timeoutInSeconds);
+    }
+  }
+
+  public static void clickIfDisplayed(WebElement element) {
+    clickIfDisplayed(element, ConstantsUtil.SMALL_TIMEOUT_SECONDS);
+  }
+
+  public static void sendKeys(WebElement element, String keys) {
```

Comment:
please add javadoc for all public methods

Reply:
Sure, they will be added as part of https://cdap.atlassian.net/browse/CDAP-18884 in a separate PR.